tests/integration_tests/scripts/tests/ixbrl-viewer_cli.bat

```diff
@@ -0,0 +1,31 @@
+@REM Download samples:

```

Comment:
Out of curiosity, did you look into using a python script to handle what's in the batch and shell script files and only using a subprocess to call the arelle command line?